recipes/kaiju/meta.yaml

```diff
@@ -1,25 +1,29 @@
+{% set version = "1.6.0" %}
+
 package:
   name: kaiju
-  version: 1.5.0
+  version: {{ version }}
 
 source:
-    fn: v1.5.0.tar.gz
-    url: https://github.com/bioinformatics-centre/kaiju/archive/v1.5.0.tar.gz
-    md5: e8effb834a7d1e2863880c60d481ecf2
-
+    fn: v{{ version }}.tar.gz
+    url: https://github.com/bioinformatics-centre/kaiju/archive/v{{ version }}.tar.gz
+    sha256: ac15caff643bd6edf2a7a5f9187aab8d1b114d9bc2e06daa97b8ac741a0c7b1b
 
 build:
   number: 0  
   no_link:
-    - bin/makeDB.sh
+    - bin/makeDB.sh -e
 
 requirements:
   build:
     - gcc
+    - zlib {{ CONDA_ZLIB }}*
+
   run:
     - libgcc
     - perl
     - gnu-wget >=1.16
+    - zlib {{ CONDA_ZLIB }}*
 
 test:
   commands:
```

Comment:
Can you change/simplify the test command to 
```yaml
    - kaiju 2>&1 | grep "Kaiju {{ version }}"
```
? This is a common workaround for such cases, hence you can then also remove the comment below.